olive/passes/onnx/conversion.py

```diff
@@ -117,7 +117,22 @@ def _run_for_config(
             # only handle dict for now since we cannot get the name of the input from a list/tuple
             if isinstance(dummy_inputs, dict):
                 dummy_input_keys = set(dummy_inputs.keys())
+
+                # handle dummy inputs for hf model with past, which has past_key_values
+                # match input names in `past_key_values.(hidden_layer_num).(key|value)` pattern
+                from transformers.modeling_utils import PreTrainedModel
```

Comment:
add try/catch since user might not install transformers.

Reply:
`transformers` is a requirement for olive so this won't happen. Plus the new change is throwing importerror which it would do regardless